docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+McClung Museum, we are still technically the creators of the records in these situations. Despite this, we typically list
+these institutions as the record creator because we set up <recordContentSource> as the element that DPLA should map to
+for content provider. In actuality, when the content provider is not UTK, this information should be communicated in
+<physicalLocation> and our DPLA mapping should be updated. Despite these semantic issues, UTK has consistently put this
+information in an incorrect element, so the mapping is not affected. In actuality then, the mapping below is for the
+content provider or institution that would be present in <physicalLocation> if MODS was followed more closely.
+
+Justification
+^^^^^^^^^^^^^
+
+A content provider is required in DPLA. This value also provides UTK with the opportunity to attribute collections to
+the institution that provided them, which is important for maintaining respectful relationships.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordContentSource"
+
+Decision
+^^^^^^^^
+
+Note that the mappings below assume that DPLA will be able to handle both literals and URIs for edm:dataProvider and edm:provider,
+as is allowed by these properties. This needs to be confirmed before this mapping is used. edm:dataProvider is used instead of
+edm:provider when the institution listed is not UTK as it  is defined as "The name or identifier of the organisation who
+contributes data indirectly to an aggregation service." Because UTK acts as a service hub for DPLA and it delivers data
+directly to this aggregator, it can be considered an edm:provider. This is defined as "The name or identifier of the
+organization who delivers data directly to an aggregation service (e.g. Europeana)."
+
+When UTK physically holds the material and created the record, the metadata resembles this `example record - acwiley:284 <https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A284/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource valueURI="http://id.loc.gov/authorities/names/n87808088">University of Tennessee, Knoxville. Libraries</recordContentSource>
+    </recordInfo>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider <http://id.loc.gov/authorities/names/n87808088> .
+
+Note that items held by UTK can also appear without a URI defining the institution as with `tdh:8803 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A8803/datastream/MODS/view>`_.
+It looks like we also have a misspelling here.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordContentSource>University of Tennesse Knoxville. Libraries</recordContentSource>
+    </recordInfo>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+        <https://example.org/objects/1> edm:provider <University of Tennesse Knoxville. Libraries> .
```

Comment:
Thanks Mark - I just caught that myself and changed it. I messaged because I didn't want people to dig too far into this tonight since I know there are errors.